app/policies/project_policy.rb

```diff
@@ -13,6 +13,10 @@ def update?
     record.maintainers.include?(user)
   end
 
+  def edit?
```

Comment:
Ebben a PR-ban nem került be új endpoint. Ez alapján nincs szükségünk új policy függvényre.

Az implementáció alapján pedig, tényleg nincs szükségünk itt új függvényre. Kérlek nézd át a következőket:
Ez a Policy class az ApplicationPolicy-ből szármezik le. Az ApplicationPolicy-ben ez szerepel:
`/app/policies/application_policy.rb` : `26-28`
```
  def edit?
    update?
  end
```